bittensor/synapses/dpn/model.py

```diff
@@ -17,22 +17,22 @@ class DPNSynapse(bittensor.Synapse):
     """
 
     def __init__(   self, 
-                    config: DPNConfig,
+                    config: DPNConfig = None,
                     dendrite: bittensor.Dendrite = None,
                     metagraph: bittensor.Metagraph = None
                 ):
         r""" Init a new DPN synapse module.
 
             Args:
-                config (:obj: `bittensor.dpn.dpn_configuration.DPNConfig`, `required`)
+                config (:obj: `bittensor.dpn.dpn_configuration.DPNConfig`, `required`, defaults to DPNConfig())
```

Comment:
`bittensor.dpn.dpn_configuration.DPNConfig` should be `bittensor.dpn.config.DPNConfig`